### PR TITLE
Should fix Notes

### PR DESCRIPTION
**Pull request by JamieD1**
#### Changelog

:cl:  
tweak: Should fix notes
/:cl:

### CODE REVIEW

File: yogstation/code/modules/client/client_procs.dm [hunk 1]
```diff
@@ -9,8 +9,6 @@
 	if(!query_logout.Execute())
 		log_sql("Failed: [query_logout.last_error]")
 	log_sql("SQL: [query_logout.sql]")
```
Comment (Bibby0110) [inline yogstation/code/modules/client/client_procs.dm]:
```suggestion
```

File: yogstation/code/modules/client/client_procs.dm [hunk 2]
```diff
@@ -9,8 +9,6 @@
 	if(!query_logout.Execute())
 		log_sql("Failed: [query_logout.last_error]")
```
Comment (Bibby0110) [inline yogstation/code/modules/client/client_procs.dm]:
```suggestion
```

File: yogstation/code/modules/client/client_procs.dm [hunk 3]
```diff
@@ -9,8 +9,6 @@
 	if(!query_logout.Execute())
```
Comment (Bibby0110) [inline yogstation/code/modules/client/client_procs.dm]:
```suggestion
	query_logout.Execute()
```

File: yogstation/code/modules/admin/sql_message_system.dm [hunk 4]
```diff
@@ -10,7 +10,7 @@
 		if(!new_key)
 			return
 		var/new_ckey = ckey(new_key)
-		var/datum/DBQuery/query_find_ckey = SSdbcore.NewQuery("SELECT ckey FROM [format_table_name("player")] WHERE ckey = :new_ckey", list("new_key" = new_ckey))
+		var/datum/DBQuery/query_find_ckey = SSdbcore.NewQuery("SELECT ckey FROM [format_table_name("player")] WHERE ckey = :new_ckey", list("ckey" = new_ckey))
```
Comment (TheGamerdk) [inline yogstation/code/modules/admin/sql_message_system.dm:13]: This won't work.

Comment (Bibby0110) [inline yogstation/code/modules/admin/sql_message_system.dm]:
```suggestion
		var/datum/DBQuery/query_find_ckey = SSdbcore.NewQuery("SELECT ckey FROM [format_table_name("player")] WHERE ckey = :new_ckey", list("new_ckey" = new_ckey))
```

**Pull request merged**